File: unit-testing-php/src/Movie.php
<?php
namespace PhpUnitWorkshop;

class Movie
{
    public const NEW_RELEASE = 'NEW_RELEASE';
    public const REGULAR = 'REGULAR';
    public const CHILDREN = 'CHILDRENS';

    private $title;

    private $type;

    public function __construct(string $title, string $type)
    {
        $this->title = $title;
        $this->type = $type;
    }

    public function getTitle(): string
    {
        return $this->title;
    }

    public function getType(): string
    {
        return $this->type;
    }

    public function isNewRelease(): bool
    {
        return $this->type === static::NEW_RELEASE;
    }
}

File: kata/kata-videostore-cs/VideoStore/Movie.cs
using System;

namespace VideoStore
{

    public enum MovieCategory
    {
        Childrens,
        Regular,
        NewRelease
    }
    public class Movie
    {

        public string Title { get; }
        public MovieCategory Category { get; }

        public Movie(string title, MovieCategory category)
        {
            Title = title;
            Category = category;
        }
    }
}